### ISSUE TITLE
Error when quit vim

### ISSUE THREAD

**Issue by PhyX-Meow** [kdheepak/JuliaFormatter.vim#33]
- vim version: 8.2-2891
- JuliaFormatter.vim: d30ba16e1f5236f1ec58e1dc1b778c77ab23230f

Hello, today after I update the plugin, these errors occur whenever I quit vim from arbitray file.

![image](https://user-images.githubusercontent.com/17513581/123508377-9b637f80-d6a1-11eb-98ba-ee0343dba1cb.png)

Translation: 
Error when processing autocmd "*" ..function JuliaFormatter#kill:
Line 6:
E475: Invalid argument: 0
Line 7:
E475: Invalid argument: 0

**Comment by kdheepak** [comment]
Can you try the most recent version of the repo? I believe this is fixed.

**Comment by PhyX-Meow** [comment]
@kdheepak Unfortunately, it still gives me error. 😥

**Comment by kdheepak** [comment]
Can you tell me what the error is exactly? I’m not able to tell from your screenshot.

**Comment by PhyX-Meow** [comment]
@kdheepak All what I got is in the screenshot, is language the problem?

**Comment by PhyX-Meow** [comment]
<img width="715" alt="119" src="https://user-images.githubusercontent.com/17513581/123737492-9c7cf280-d8d5-11eb-8f4d-160e6dd25a07.png">

Reproduce:
Open vim,
Press `:q<CR>`.

**Comment by PhyX-Meow** [comment]
Mininal `.vimrc`:
```
call plug#begin('~/.vim/plugged')
Plug 'JuliaEditorSupport/julia-vim'
Plug 'kdheepak/JuliaFormatter.vim'
  vnoremap <localleader>jf :<C-u>call JuliaFormatter#Format(1)<CR>
  nnoremap <localleader>jf :<C-u>call JuliaFormatter#Format(0)<CR>
call plug#end()
```

**Comment by kdheepak** [comment]
Surprisingly I’m not getting this error. I’ll try with a fresh instance of nvim in the morning.

**Comment by kdheepak** [comment]
Oh you are using vim! I didn’t check that this works with vim. I’ll make the PR in the morning.

**Comment by kdheepak** [comment]
Hmm, I just tried it and I'm not getting the error with vim, using the following `MINRC`.

```
" If after installing JuliaFormatter.vim, you are having trouble getting it to work, try the following minimal vimrc file.

set nocompatible              " be iMproved, required
filetype off                  " required

call plug#begin('~/.vim/plugged')

Plug '~/gitrepos/JuliaFormatter.vim'

" Initialize plugin system
call plug#end()

nnoremap <space>jf :<C-u>call JuliaFormatter#Format(0)<CR>
vnoremap <space>jf :<C-u>call JuliaFormatter#Format(1)<CR>

" Save the above to a file `MINRC`, and run `vim -u MINRC tests/test.jl` and try hitting `\jf` in normal or visual mode.

nnoremap <silent> <space>sv :source tests/MINRC<CR>
```


https://user-images.githubusercontent.com/1813121/123755996-b5f05100-d879-11eb-90af-f59817c2f55d.mov

**Comment by PhyX-Meow** [comment]
Seems something go wrong in plugin/JuliaFormatter.vim, the server should be launched but didn't, and in this case `JuliaFormatter#Kill()` should not be called but it did.

**Comment by kdheepak** [comment]
I tried just opening vim and closing it, and I still didn’t get the error. Maybe I’ll try with the latest version of vim today.

**Comment by kdheepak** [comment]
Can you try it with this change: https://github.com/kdheepak/JuliaFormatter.vim/commit/c18ddb3b8b4425bae5ec1781e6ad74fb79d5e875.

**Comment by kdheepak** [comment]
I'm pretty sure this fixed it. While I was not able to reproduce the issue using the latest version of `vim` (which is so surprising), I was able to figure out what was going on. `job_status` and `job_stop` in `vim` does not accept integers as an argument and expects a `job` object which can only be created by `job_start`. This check will solve it. I tested it by calling the kill function directly i.e. `:call JuliaFormatter#Kill()`.
